crates/cairo-lint-core/tests/tests.rs

```diff
@@ -453,3 +453,12 @@ test_file!(
     "int le min one",
     "int lt min one"
 );
+
+test_file!(
+    redundant_op,
+    redundant_op,
+    "multiplication by one",
+    "addition by zero",
+    "division by one",
+    "substraction by zero"
+);
```

Comment:
Today we merged new tests model. Please adjust to the new changes. There's not a lot of them, but still